src/core/Map/types/index.ts

```diff
@@ -82,6 +83,13 @@ export type EngineProps = {
   onTick?: (clock: Date) => void;
   onLayerDrag?: (layerId: string, position: LatLng) => void;
   onLayerDrop?: (layerId: string, propertyKey: string, position: LatLng | undefined) => void;
+  onLayerEdit?: (e: LayerEditEvent) => void;
```

Comment:
Change `Map/index.tsx` to pass `onLayerEdit` prop to `Engine` component.

Reply:
I've fixed this comment, and the bellow comment has also been fixed.
[fix: pass onLayerEdit prop to context](https://github.com/reearth/reearth-web/pull/377/commits/86db7c03000ca3ddf3c4094b7593fae5586b4b2f)